src/logging/LogUtil.ts

```diff
@@ -2,6 +2,9 @@ import { LazyLoggerFactory } from './LazyLoggerFactory';
 import type { Logger } from './Logger';
 import type { LoggerFactory } from './LoggerFactory';
 
+let loggerFactoryWrapper = new LazyLoggerFactory();
+let classLoggers = new WeakMap<Constructor, Logger>();
+
 /**
  * Gets a logger instance for the given class instance.
  *
```

Comment:
There is an `{@link Setup}` on the next line which I think should link to the `AppRunner` now.